main.tf

```diff
@@ -82,11 +82,14 @@ resource "ibm_database" "postgresql_db" {
     }
   }
 
+  # Workaround for https://github.ibm.com/GoldenEye/issues/issues/11359
+  # means that no `group` block is added when restoring from backup
+
   ## This for_each block is NOT a loop to attach to multiple group blocks.
   ## This is used to conditionally add one, OR, the other group block depending on var.local.host_flavor_set
   ## This block is for if host_flavor IS set to specific pre-defined host sizes and not set to "multitenant"
   dynamic "group" {
-    for_each = local.host_flavor_set && var.member_host_flavor != "multitenant" ? [1] : []
+    for_each = local.host_flavor_set && var.member_host_flavor != "multitenant" && var.backup_crn == null ? [1] : []
```

Comment:
Seems this may prevent scaling after restoring from a backup?

Reply:
The restore from backup is a one time operation.

In order to scale after restoring from backup, the process should (must) pass null in the backup_crn. This would restore the group block (per previous behaviour).